otvali_bot/models/__init__.py

```diff
@@ -0,0 +1,10 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, scoped_session
+from config.bot import sql_engine_url
+
+engine = create_engine(sql_engine_url, echo=False, connect_args={'check_same_thread': False})
```

Comment:
remove chech_same_thread attribute. because of it the bot can't connect to the postgres